scripts/check

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Checks if the dependencies for each service exists
+# Kubernetes Dependencies
+command -v kubectl >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency kubectl does not exist"
+}
+command -v minikube >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency minikube does not exist"
+}
+command -v docker >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency docker does not exist"
+}
```

Comment:
Just a suggestion but can't you turn this into a function like 
`does_cmd_exist() {
  command -v $1 > /dev/null 2>&1
}` and then call it for each dependency that we need to install by calling `does_cmd_exist kubectl` or something

Reply:
Created does_cmd_exist